workshops/personal_website/README.md

```diff
@@ -163,331 +122,204 @@ Every HTML file follows this template:
     </body>
 </html>
 ```
+This structure is common to all HTML pages. If you want to find out more, just
```

Comment:
> If you want to find out more

I feel like this implies that by looking at any web page's source, it'll tell you why this structure exists. Maybe something more like "This structure is common to all HTML pages. In fact, you can take a look for yourself! Just right click on any web page, including this one, and click "View source" to see what's going on behind the scenes."